### ISSUE TITLE
Error: Swift Compiler Error (Xcode): Call to method 'getAppGroupDirectory'

### ISSUE THREAD

**Issue by NomiAdam** [istornz/flutter_live_activities#99]
Just tried the latest version and for me the xCode build fails on this error 

```
Swift Compiler Error (Xcode): Call to method
'getAppGroupDirectory' in closure requires explicit use
of 'self' to make capture semantics explicit
/Users/adamkvasnicka/.pub-cache/hosted/pub.dev/flutter_ap
p_group_directory-1.0.2+1/ios/Classes/FlutterAppGroupDire
ctoryPlugin.swift:20:25


Uncategorized (Xcode): Command SwiftEmitModule failed
with a nonzero exit code
```

Tried clearing caches etc. but issue still persist, something on my side or possibly an issue with latest version? 🙏

**Comment by lukeyuen701** [comment]
After updating, I encountered the same issue.

**Comment by istornz** [comment]
Got it, found the issue! I will fix it ASAP 👍

**Comment by istornz** [comment]
@NomiAdam @lukeyuen701 Should be fixed with the `2.0.0+2` version, sorry for the bug :/

**Comment by lukeyuen701** [comment]
> @NomiAdam @lukeyuen701 Should be fixed with the `2.0.0+2` version, sorry for the bug :/

It works! Thank you!

**Comment by NomiAdam** [comment]
@istornz Legend! Thanks man ❤️
